language/move-lang/src/bin/move-check.rs

```diff
@@ -44,6 +44,15 @@ pub struct Options {
         long = cli::OUT_DIR,
     )]
     pub out_dir: Option<String>,
+
+    /// If set, allow modules defined in source_files to shadow modules of the same id that exist
+    /// in dependencies
+    #[structopt(
+        name = "SOURCES_SHADOW_DEPS",
+        short = cli::SHADOW_SHORT,
+        long = cli::SHADOW,
+    )]
+    pub shadow: bool,
```

Comment:
can you set the default to be true? 

I feel like it is more intuitive than the alternative. But thoughts?

Reply:
I don't have a strong opinion either way.

Reply:
Made an update to default it to be true.